prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -853,6 +886,18 @@ ssize_t rxr_pkt_proc_matched_rtm(struct rxr_ep *ep,
 		rx_entry->cq_entry.len = rx_entry->total_len;
 
 	pkt_type = rxr_get_base_hdr(pkt_entry->pkt)->type;
+
+	if (pkt_type >= RXR_DC_REQ_PKT_BEGIN &&
+	    pkt_type <= RXR_EXTRA_REQ_PKT_END)
```

Comment:
No, you cannot assume all extra feature is DC. We will very likely add new packet types in the future.

Reply:
Using `RXR_DC_REQ_PKT_END `, resolving.